releases/v22_0_0.rst

```diff
@@ -0,0 +1,255 @@
+
+Initial integration of Piff
+---------------------------
+`Piff <https://github.com/rmjarvis/Piff>`_ has been added to rubin-env as a third party package, and the integration package `meas_extension_piff`_ has been added to lsst_distrib.
+`meas_extension_piff`_ integrates Piff with the pipelines.
+Tasks can be configured to use Piff PSF models instead of the current default PSFex.
+:jirab:`RFC-755`
+
+.. _meas_extension_piff: https://github.com/lsst/meas_extensions_piff
+
+.. _release-v22-0-0-scarlet:
+
+Coadd measurement now uses the Scarlet deblender by default
+-----------------------------------------------------------
+`Scarlet<https://pmelchior.github.io/scarlet/>` is now the default deblender used for measurements on coadds.
+The configuration for single-frame measurement has not changed.
+Scarlet produces additional flags for filtering duplicates, as it handles isolated sources differently: :code:`deblend_parentNPeaks`, :code:`deblend_parentNChild`.
+:code:`detect_isPrimary` is still populated, and is the recommended column.
+See the `community.lsst.org post <https://community.lsst.org/t/new-use-of-isprimary-that-began-in-w-2021-12/4957/2>`_ for more information.
+:jirab:`RFC-745`
+
+.. _release-v22-0-0-dustmaps:
+
+Dust Maps
+---------
+The `dustmaps_cachedata <https://github.com/lsst/dustmaps_cachedata>`_ package has been added to the pipelines, and the `dustmaps <https://dustmaps.readthedocs.io/>`_ module is a third-party dependency.
+:jirab:`RFC-752`
+
+.. _release-v22-0-0-faro:
+
+:ref:`Faro <lsst.faro.base>` package for scientific performance metrics
+-----------------------------------------------------------------------
+A package for measuring scientific performance metrics, :ref:`faro <lsst.faro.base>`, was added.
+It implements the Gen3 versions of the tasks in :ref:`validate_drp <lsst.validate.drp>`. See :ref:`release-v22-0-0-deprecate-validate-drp`.
+This package was used to generate the Characterization Report for this release.
+:jirab:`RFC-753, DM-28351`
+
+
+
+.. _release-v22-0-0-interface:
+
+Significant Interface Changes
+=============================
+
+- :ref:`release-v22-0-0-filterLabel`
+- :ref:`release-v22-0-0-remove-obs_ctio0m9`
+- :ref:`release-v22-0-0-remove-metric-in-commonMetrics.py`
+
+
+.. _release-v22-0-0-filterLabel:
+
+Replace afw.image.Filter with FilterLabel
+-----------------------------------------
+Filter information is now stored as a FilterLabel, replacing the existing `~lsst.afw.image.filter`.
+This class replaces `~Filter`'s system of names, canonical names, and aliases with just two names: a `~band` (e.g., “g” or “r”) and a `~physical filter` (e.g., “g DECam SDSS c0001 4720.0 1520.0” or “HSC-R2”).
+Note that not all FilterLabel objects have both a band and a physical filter, especially during the transition period, so please program defensively.
+See `community.lsst.org <https://community.lsst.org/t/migrating-from-afw-image-filter-to-filterlabel/459>`__ for details.
+:jirab:`RFC-730`
```

Comment:
Code literals and links are getting mixed-up/broken there. `lsst.afw.image.filter` Isn’t resolving to a link (the reason to use single backticks), or is Filters resolving. I’d recommend either:

- putting all the API-ish things here that are in single backticks into double backticks and dropping the `~`
- working to see if if the API links can be resolved. Since FilterLabel is a new API, right, I think that link would be useful.